data/event_scripts.s

```diff
@@ -55,16 +60,16 @@ gSpecialVars:: @ 815FD0C
 
 	.align 2
 gStdScripts:: @ 8160450
-	.4byte gStdScript_0
-	.4byte gStdScript_1
-	.4byte gStdScript_2
-	.4byte gStdScript_3
-	.4byte gStdScript_4
-	.4byte gStdScript_5
-	.4byte gStdScript_Message_WaitButton_AndRelease
-	.4byte gStdScript_7
-	.4byte gStdScript_8
-	.4byte gStdScript_ItemMessageAndFanfare
+	.4byte Std_ObtainItem
+	.4byte Std_FindItem
+	.4byte Std_MsgboxNPC
+	.4byte Std_MsgboxSign
+	.4byte Std_MsgboxDefault
+	.4byte Std_MsgboxYesNo
+	.4byte Std_MsgboxAutoclose
+	.4byte Std_ObtainDecoration
+	.4byte Std_PutItemAway
+	.4byte Std_ReceivedItem
```

Comment:
Why is the tense different here? (Received - Obtain)

Reply:
Because the tense of the action is different. Obtain is used to obtain items, received item just displays a received item message for after an item is given.